### PR TITLE
Chore: Warn by default if the error occurs during the schema creation

### PR DESCRIPTION
**Pull request by izeigerman**

### CODE REVIEW

File: sqlmesh/core/engine_adapter/base.py [hunk 1]
```diff
@@ -456,15 +456,22 @@ def create_view(
...
+                exp.Create(
+                    this=exp.to_identifier(schema_name.split(".")[0]),
+                    kind="SCHEMA",
+                    exists=ignore_if_exists,
+                )
             )
-        )
+        except Exception as e:
+            if not warn_on_error:
+                raise
```
Comment (GeorgeSittas) [inline sqlmesh/core/engine_adapter/base.py:473]:
```suggestion
                raise e
```

Comment (izeigerman) [reply sqlmesh/core/engine_adapter/base.py:473]: why?

Comment (izeigerman) [reply sqlmesh/core/engine_adapter/base.py:473]: See https://stackoverflow.com/questions/18001721/raise-with-no-argument

> When you raise without arguments, the interpreter looks for the last exception raised and handled. It then acts the same as if you used raise with the most recent exception type, value and traceback.

Comment (GeorgeSittas) [reply sqlmesh/core/engine_adapter/base.py:473]: TIL! Thanks for sharing -- ignore me then.

**Pull request merged**